yb-voyager/cmd/importData.go

```diff
@@ -1035,8 +1062,15 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 			// creating fresh connection
 			(*conn).Close(context.Background())
 			*conn = newTargetConn()
+
+			// Extract the schema name and add to the index name
+			fullyQualifiedObjName, err := getIndexName(sqlInfo.stmt, sqlInfo.objName)
+			if err != nil {
+				utils.ErrExit("unable to get index name qualified with schema name for DDL %v: %w", sqlInfo.stmt, err)
```

Comment:
Be precise and concise about the messages:

    "extract qualified index name from DDL [%v]: %w", sqlInfo.stmt, err

Reply:
Done